lib_version.clj

```diff
@@ -1,4 +1,4 @@
 (ns lib-version)
 
-(def lib 'district0x/district-ui-component-active-account) ; ends up as <group-id>/<artifact-id> in pom.xml
+(def lib 'io.github.district0x/district-ui-component-active-account) ; ends up as <group-id>/<artifact-id> in pom.xml
```

Comment:
Why do you want to publish under a longer _group id_ ? We can still update existing libraries under `district0x` e.g. https://clojars.org/district0x/district-ui-component-active-account